### ISSUE TITLE
Alt text is read as "button" by screen reader, not "image" as expected

### ISSUE THREAD

**Issue by hs4man21** [bluesky-social/social-app#969]
**Describe the bug**
With the recent update, the screen reader VoiceOver is reading alt text values as buttons, which is confusing to screen reader users. I suppose they are a button in the sense that the image can be expanded, but the `accessibilityRole` should be image. Maybe this is a case where an `accessibilityHint` can be used to explain that the images can be clicked to be expanded. 

Something like:
```
accessibilityRole="image"
accessibilityHint="Click to expand image"
```

(Expanding images can still be relevant since not everyone who uses a screen reader is blind)

**To Reproduce**

1. Turn on VoiceOver. (Testing manually with VoiceOver is important! It's actually very beginner-friendly: https://support.apple.com/guide/iphone/turn-on-and-practice-voiceover-iph3e2e415f/ios#:~:text=Activate%20Siri%20and%20say%20“Turn,the%20setting%20on%20or%20off)
2. Position VoiceOver on any image in the BlueSky mobile app by tapping on one with your finger.
3. Listen to the description that is read by VoiceOver.

**Expected behavior**

VoiceOver should describe the image as an "image," not as a "button." After a pause, it should say something like "Click to expand image."

**Details**

- Platform: iPhone XR, iOS 16.4.1, using VoiceOver screen reader
- App version: 1.38

**Comment by pfrazee** [comment]
Got it! Appreciate it -- again, something I'm wrapping my head around here
